src/ray/protobuf/experimental/instance_manager.proto

```diff
@@ -135,6 +135,34 @@ message LaunchRequest {
   int64 request_ts_ms = 4;
 }
 
+// Termination request for an instance.
+message TerminationRequest {
```

Comment:
Should we name it StopRequest to match the state machine?

Reply:
will update in seperate pr to change the state machine from STOPPING to TERMINATING

Reply:
https://github.com/ray-project/ray/pull/42838